src/main/kotlin/step2Lotto/domain/dto/LottoRank.kt

```diff
@@ -0,0 +1,9 @@
+package step2Lotto.domain.dto
+
+enum class LottoRank(val prizeMoney: Int) {
+    FIRST(2000000000),
+    THIRD(1500000),
+    FOURTH(50000),
+    FIFTH(5000),
+    LOSE(0)
```

Comment:
큰 숫자의 가독성 확보를 위해 `_`를 붙여줄 수 있어요🙂
```suggestion
    FIRST(2_000_000_000)
    ...
```